selfdrive/ui/linux.cc

```diff
@@ -64,6 +64,8 @@ void framebuffer_swap(FramebufferState *s) {
   glfwPollEvents();
 }
 
+bool set_brightness(int brightness) { return true; }
```

Comment:
Why do we need this implementation? The other one will just not do anything on a computer.

Reply:
framebuffer.cc will not be built on computer, this dummy implementation is used to avoid the link error.